test/models/micropost_test.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+# Micropost model test class. 
+class MicropostTest < ActiveSupport::TestCase
+    test 'should succeed empty micropost content' do
```

Comment:
2 space indents. Check out our model tests in IO and DM. e.g. https://github.com/IntouchHealth/IO/blob/master/test/models/file_resource_test.rb and structure it like those.